openwisp_users/migrations/0003_default_organization.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.1.1 on 2018-09-16 12:20
+
+from django.db import migrations
+from django.conf import settings
+
+
+def set_default_organization_uuid(apps, schema_editor):
+    """
+    Get default organzation or create a default
+    organization and set settings.DEFAULT_ORG_UUID
+    """
+    organization = apps.get_model('openwisp_users', 'organization')
+    try:
+        default_organization = organization.objects.get(
+            slug=settings.DEFAULT_ORG_SLUG)
```

Comment:
@nemesisdesign 
For the case of user having multiple organizations,
it might be good to provide an option to enter the organization that user feels is appropriate to be default, hence by using `DEFAULT_ORG_SLUG`, the user can define the default organization.

Is this possibly useful or should i set default to the first organisation i get from database? Please suggest! :smile:

Reply:
as in the previous comment, the organization should be created only if it doesn't already exist.
I'd rename the temporary setting to `_OPENWISP_DEFAULT_ORG_SLUG`, the leading underscore indicating it's a python convention to indicate an internal thing and shouldn't be relied upon.